src/react/databrowser/buckets/details/__tests__/Overview.test.jsx

```diff
@@ -0,0 +1,135 @@
+        expect(secondContentItem.find(T.Key).text()).toContain('Versioning');
+        expect(secondContentItem.find(Toggle).text()).toContain('Suspended');
+
+        let thirdContentItem = firstGroupInfosContentItems.at(2);
+        expect(thirdContentItem.find(T.Key).text()).toContain('Location');
+        expect(thirdContentItem.find(T.Value).text()).toContain('us-east-1');
+
+        // SECOND GROUP ITEMS TITLE
+        const secondGroupInfos = groupInfos.at(1);
+        expect(secondGroupInfos.find(T.GroupName).text()).toContain('Permissions');
+
+        // SECOND GROUP ITEMS
+        const secondGroupInfosContentItems = secondGroupInfos.find(T.GroupContent).find(T.Row);
+        // console.log(secondGroupInfos.find(T.GroupContent).find(T.Row).debug())
+        firstContentItem = secondGroupInfosContentItems.first();
+        expect(firstContentItem.find(T.Key).text()).toContain('Owner');
+        expect(firstContentItem.find(T.Value).text()).toContain('bart');
+
+        secondContentItem = secondGroupInfosContentItems.at(1);
+        expect(secondContentItem.find(T.Key).text()).toContain('Access Control List');
+        expect(secondContentItem.find(T.Value).text()).toContain('0 Grantee');
+
+        thirdContentItem = secondGroupInfosContentItems.at(2);
+        expect(thirdContentItem.find(T.Key).text()).toContain('CORS Configuration');
+        expect(thirdContentItem.find(T.Value).text()).toContain('No');
+
+        let fourthContentItem = secondGroupInfosContentItems.at(3);
+        expect(fourthContentItem.find(T.Key).text()).toContain('Public');
+        expect(fourthContentItem.find(T.Value).text()).toContain('No');
+    });
+
+    it('should render toggle versioning in Enable mode', () => {
+        const { component } = reduxMount(<Overview bucket={BUCKET}/>, {
+            ...TEST_STATE, ...{
+                s3: {
+                    bucketInfo: {
+                        ...BUCKET_INFO_RESPONSE, ...{
+                            versioning: 'Enable',
+                            isVersioning: true,
+                        },
+                    },
+                },
+            },
+        });
+
+        const groupInfos = component.find(T.Group);
+        const firstGroupInfos = groupInfos.first();
+
+        let versioningToggleItem = firstGroupInfos.find(Toggle);
+        expect(versioningToggleItem.text()).toContain('Enable');
+    });
+
+    it('should trigger deleteBucket function when approving clicking on delete button when modal popup', () => {
+        const deleteBucketMock = jest.spyOn(actions, 'deleteBucket');
+
+        const { component } = reduxMount(<Overview bucket={BUCKET}/>, {
+            ...TEST_STATE, ...{
+                uiBuckets: {
+                    showDelete: bucketName,
+                },
+            },
+        });
+
+
+        const buttons = component.find(DeleteConfirmation).find(Button);
+        const deleteButton = buttons.at(1);
```

Comment:
Select button by id to avoid failure if we switch the two buttons for example.

Reply:
done